### DISCUSSION TITLE
Lower-end GPU Support

### DISCUSSION THREAD

**Discussion by Oceaneyes123** [moyix/fauxpilot#121] [Q&A]
Does it work in NVIDIA GTX 1650 Super?

**Comment by moyix** [comment]
Not sure unfortunately – I would recommend trying it out! And if you find it does / does not work, please add an entry here so others can benefit :) https://github.com/moyix/fauxpilot/wiki/GPU-Support-Matrix

**Comment by viktor02** [comment]
Is it possible to quantize models in 4 bits like people do with llama to run on consumer gpu?
